sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ChangeFeedProcessorOptions.java

```diff
@@ -55,6 +60,8 @@ public ChangeFeedProcessorOptions() {
         this.leaseExpirationInterval = DEFAULT_EXPIRATION_INTERVAL;
         this.feedPollDelay = DEFAULT_FEED_POLL_DELAY;
         this.maxScaleCount = 0; // unlimited
+
+        this.scheduler = Schedulers.boundedElastic();
```

Comment:
We have dedicated schedulers for bulkExecutor, rntbd (CosmosSchedulers.cs), so I wonder should we also use a dedicated one for changeFeedProcessor.

Reply:
We could but that will probably be a different change while we can test that out. My concern with introducing our own scheduler for CFP is that we need to test it based on the configuration we choose. If it's just the name of it, so logging is easier to understand, that might be a low hanging fruit.